lib/src/super_extensions_base.dart

```diff
@@ -151,6 +151,33 @@ extension StringExtension on String {
   void copyToClipboard() => Clipboard.setData(ClipboardData(text: this));
 }
 
+extension NullableStringExtension on String? {
+  /// Extension method on String? to eliminate the need for a null-check before
+  /// checking if a string is empty, inspired by method of same name from kotlin
+  /// Sample usage:
+  /// ```dart
+  /// String? text = null;
+  /// if (text.isBlank) {
+  ///   // do something
+  /// }
+  /// ```
+  bool get isBlank {
+    if (this != null && this!.isNotEmpty) return false;
+
+    return true;
```

Comment:
This can be refactored as `return !(this!=null && this!.isNotEmpty)`

Reply:
Oh yeah. Thanks for the correction